src/WithModelQuery.ts

```diff
@@ -50,8 +50,8 @@ export function withModelQuery<P = {}>(options?: IWithModelQueryOptions & P): Co
         ),
       ),
     ),
-    typeof options !== "undefined" && options.hasOwnProperty("loadingComponent")
-      ? withLoadingIndicator<IWithModelQueryOptions>(({ items }) => items == null, options.loadingComponent)
+    typeof options !== "undefined" && options.hasOwnProperty("componentProps")
+      ? withLoadingIndicator<IWithModelQueryOptions>(({ items }) => items == null, options.loadingComponent, options.componentProps)
       : withLoadingIndicator<IWithModelQueryOptions>(({ items }) => items == null),
```

Comment:
I did initially remove this, but ended up having to put a check for the props otherwise they were never passed through.